terraform/templates/dashboards/network.json

```diff
@@ -717,7 +717,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "network_gauge{op=\"pending_network_requests\"}",
+          "expr": "network_gauge{op=\"pending_network_requests\"} > 0",
```

Comment:
what is the `> 0` for?

Reply:
Only show the lines when the value of the gauge is positive